common/src/main/scala/org/genivi/sota/refined/SprayJsonRefined.scala

```diff
@@ -29,11 +29,14 @@ trait SprayJsonRefined {
       }
     }
 
-  implicit def refinedUnmarshaller[P](implicit p: Predicate[P, String]) = Unmarshaller.strict[String, Refined[String, P]] { string =>
-    refineV[P](string) match {
-      case Left(e)  => throw new IllegalArgumentException(e)
-      case Right(r) => r
-    }
+  implicit def refinedUnmarshaller[P]
```

Comment:
I would like to ask you to avoid reformatting the code in future unless it really makes sense.